### ISSUE TITLE
'wbx' not available on Windows 10 (Python 3)

### ISSUE THREAD

**Issue by imknown** [xpirt/sdat2img#23]
https://github.com/xpirt/sdat2img/blob/c99e13b9a461129c84f481686c462a522ebc8cc8/sdat2img.py#L83

```
ValueError: must have exactly one of create/read/write/append mode
```

Works with `'wb+'`.

**Comment by krystian-wojtas** [comment]
I confirm that it didn't work also on Lubuntu 18.04.
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```

Here is error message on python 3
```
$ python3 sdat2img/sdat2img.py system.transfer.list system.new.dat system.img
sdat2img binary - version: 1.2

Android Nougat 7.x / Oreo 8.x detected!

Traceback (most recent call last):
  File "sdat2img/sdat2img.py", line 143, in <module>
    main(TRANSFER_LIST_FILE, NEW_DATA_FILE, OUTPUT_IMAGE_FILE)
  File "sdat2img/sdat2img.py", line 83, in main
    output_img = open(OUTPUT_IMAGE_FILE, 'wbx')
ValueError: must have exactly one of create/read/write/append mode
```

It works fine invoked with python 2.7 or with python 3 and suggested fix 'wb+'

**Comment by xpirt** [comment]
Ok, fixed by 1b08432247fce8037fd6a43685c6e7037a2e553a.

Closed.
